docs/Deployment.md

```diff
@@ -0,0 +1,99 @@
+# Deploying MsQuic
+
```

Comment:
i would like to see some discussion of keep-alives and when and how to configure them. i'd also like you to talk about migration support on server and how that can be enabled in conjunction with a load balancer that supports it.